src/pages/settings/Subscription/CardSection/CardSection.tsx

```diff
@@ -61,13 +74,45 @@ function CardSection() {
                                     })}
                                 </Text>
                             </View>
+                            <CardSectionActions />
                         </View>
-                        <CardSectionActions />
-                    </>
+                    )}
+                    {isEmptyObject(defaultCard?.accountData) && <CardSectionDataEmpty />}
+                </View>
+                {!isEmptyObject(defaultCard?.accountData && plan && isEligibleForRefund) && (
+                    <MenuItem
+                        shouldShowRightIcon
+                        icon={Expensicons.Bill}
+                        iconStyles={[]}
+                        wrapperStyle={styles.sectionMenuItemTopDescription}
+                        style={styles.mt5}
+                        title={translate('subscription.cardSection.requestRefund')}
+                        titleStyle={styles.textStrong}
+                        onPress={() => setIsRequestRefundModalVisible(true)}
```

Comment:
Is the action available while being offline ?

Reply:
As we are using Pattern C in that flow , it shouldnt be available

Reply:
Then let's add `disabled={isOffline}` ?